client/app/components/EditParameterSettingsDialog.jsx

```diff
@@ -86,13 +86,13 @@ function EditParameterSettingsDialog(props) {
 
   // fetch query by id
   useEffect(() => {
-    const { queryId } = props.parameter;
+    const queryId = props.parameter.queryId;
```

Comment:
Is this change needed for eslint to detect the correct value for the dependencies array?

Reply:
Yes, using it like this it automatically corrects to `[props.parameter.queryId]`